Services/Simulations.cs

```diff
@@ -284,11 +284,6 @@ public async Task<Models.Simulation> MergeAsync(SimulationPatch patch)
             if (patch.Enabled == false)
```

Comment:
If the simulation was already stopped, and an PATCH request came in that didn't change Enabled, we'd be updating the StoppedTime here incorrectly. Perhaps this should be:

```suggestion
               if(patch.Enabled == false && item.Enabled == false)
```

Reply:
Looks unrelated to the changes in this PR. Could you pls open a separate bug to look into this issue ?